internal/app/app.go

```diff
@@ -27,20 +31,30 @@ func Run() {
 		os.Exit(1)
 	}
 
-	logger := setup.GetLoggerInstance()
+	logger := setup.GetLogger()
 
 	logger.Info("Logs path: ", filepath.Join(cfg.Additional.LogDir, cfg.Additional.LogFileName))
 
 	// Database
 	logger.Info("Database URI: ", cfg.DatabaseURI())
 
-	db, err := postgres.NewPostgresDatabase(cfg.DatabaseURI())
+	dbConn, err := pdb.NewConnection(cfg.DatabaseURI())
 	if err != nil {
 		logger.Fatalw("Failed connecting to database:", err)
 	}
 	logger.Info("Connected to DB")
 
-	// // TODO: Initialize services
+	db, err := gorm.Open(postgres.New(postgres.Config{
+		Conn: dbConn,
+	}), &gorm.Config{})
+	if err != nil {
+		logger.Error("Can't connect to database", err)
+	}
+
+	if err := setup.MigrateDB(db); err != nil {
```

Comment:
Мне кажется для инициализации будет лучше использовать.
db.AutoMigrate(...)

Reply:
> NOTE: AutoMigrate will create tables, missing foreign keys, constraints, columns and indexes. It will change existing column’s type if its size, precision, nullable changed. It WON’T delete unused columns to protect your data.

** Мне кажется то что нужно

Reply:
Было бы круто иметь версии миграций(в горме нет их), поэтому внутри MigrateDB() описываются миграции и ролбэки на случай ошибки, а затем вызывается db.AutoMigrate()